### PR TITLE
Longer delays for dynamic injections

### PR DESCRIPTION
**Pull request by ShiftyRail**
I'm on a roll of shitty < 10 lines changes
Open for discussion and tweaking. Maybe they're too big.

Raise the delays for dynamic injections.

Latejoin : between 5 and 17 minutes to between 10 and an hour.
Midround : between 10 and 35 minutes to between 20 and 1 hour 40 minutes.

Maybe the higher limits are too high, I don't know.

### CODE REVIEW

**Comment by Kurfursten** [comment]
Conflicts

File: __DEFINES/role_datums_defines.dm [hunk 1]
```diff
@@ -199,3 +199,11 @@
 
 #define INTERCEPT_TIME_LOW 10 MINUTES
 #define INTERCEPT_TIME_HIGH 18 MINUTES
+
+// -- Injection delays (in ticks, ie, * 2 byond seconds)
+
+#define LATEJOIN_DELAY_MIN 600
```
Comment (Kurfursten) [inline __DEFINES/role_datums_defines.dm]: use MINUTES

**Comment by ShiftyRail** [comment]
I have a different idea in how to do this.

**Comment by ShiftyRail** [comment]
Fixed the conflicts, lowered the numbers (I made them twice as big as I intended) and instead of being a random number between A and B, it's heavily weighted towards A. 

That way, B can be very long without having too much risk of the latejoin being several hours.

**Pull request merged**